rawterm.lua

```diff
@@ -24,18 +24,32 @@
 
 ]]
 
-if not jit then
-    error("The RawTerm library requires LuaJIT FFI")
-end
-
-local ffi = require("ffi")
-local band, bnot, bor = bit.band, bit.bnot, bit.bor
+local bitop = jit and
+	bit or require("bit32")
+
+local ffi = require(
+	jit and
+	"ffi" or "cffi"
+)
+local band, bnot, bor = bitop.band, bitop.bnot, bitop.bor
 local floor = math.floor
 
 local rawterm = {}
 
+local winsize = jit and [[
+unsigned short int ws_row;
+unsigned short int ws_col;
+unsigned short int ws_xpixel;
+unsigned short int ws_ypixel;
+]] or [[
+unsigned int ws_row;
+unsigned int ws_col;
+unsigned int ws_xpixel;
+unsigned int ws_ypixel;
+]]
```

Comment:
Why is the cdef different between the luajit ffi and cffi-lua?

Reply:
`unsigned short int` is invalid in cffi-lua, but valid in LuaJIT's `ffi` library

Reply:
Doesn't the definition need to be a short though? Should the definition be altered to:
```
unsigned short ws_row;
unsigned short ws_col;
unsigned short ws_xpixel;
unsigned short ws_ypixel;
```